spec/lib/postzord/receiver/local_batch_spec.rb

```diff
@@ -90,4 +90,19 @@
       receiver.perform!
     end
   end
+
+  context "received reshare" do
+    before do
+      @reshare = FactoryGirl.create(:reshare, root:
+                                              FactoryGirl.build(:status_message, author: bob.person, public: true))
+      @root = @reshare.root
+    end
+
+    let(:receiver) { Postzord::Receiver::LocalBatch.new(@object, @ids) }
+
+    it "creates a participation on the reshare for the root author" do
+      receiver
+      expect(@root.resharers.count).to eq(1)
```

Comment:
Sorry my bad, was waiting for input on how to write the other test, should not have committed this as a stub of copied from above.